files/setup-05-knative.sh

```diff
@@ -55,4 +57,23 @@ SOCKEYE_CONFIG=/root/config/knative/sockeye.yaml
 # Apply YTT overlay
 ytt --data-value-file bom=${VEBA_BOM_FILE} -f ${SOCKEYE_TEMPLATE} > ${SOCKEYE_CONFIG}
 
-kubectl -n vmware-functions apply -f ${SOCKEYE_CONFIG}
\ No newline at end of file
+kubectl -n vmware-functions apply -f ${SOCKEYE_CONFIG}
+
+# Install Tanzu Sources for Knative
+echo -e "\e[92mDeploying Tanzu Sources for Knative ..." > /dev/console
+TANZU_SOURCES_TEMPLATE=/root/download/tanzu-sources/templates/tanzu-sources-release.yaml
+TANZU_SOURCES_OVERLAY=/root/download/tanzu-sources/overlay.yaml
+TANZU_SOURCES_CONFIG=/root/download/tanzu-sources-release.yaml
+ytt --data-value-file config=${VEBA_CONFIG_FILE} -f ${TANZU_SOURCES_OVERLAY} -f ${TANZU_SOURCES_TEMPLATE} > ${TANZU_SOURCES_CONFIG}
+kubectl apply -f ${TANZU_SOURCES_CONFIG}
+
+# Wait for all controllers/webhook to be ready
+kubectl wait --for=condition=ready pod -l app=horizon-source-controller --timeout=${KUBECTL_WAIT} -n vmware-sources
+kubectl wait --for=condition=ready pod -l app=horizon-source-webhook --timeout=${KUBECTL_WAIT} -n vmware-sources
+kubectl wait --for=condition=ready pod -l app=vsphere-source-webhook --timeout=${KUBECTL_WAIT} -n vmware-sources
+
+# Enable debug logs for tanzu sources
+if [ ${TANZU_SOURCES_DEBUG} -eq "True" ]; then
+    kubectl -n vmware-sources get cm config-logging -o yaml > /tmp/vmware-sources-config-logging.yaml
+    sed -i 's/"level": "info"/"level": "debug"/g' /tmp/vmware-sources-config-logging.yaml
```

Comment:
Is this used/applied anyhwere? Not sure I'm understanding what this is supposed to do.

Reply:
🦅 👁️ No, this was temp way of enabling debugging w/sources. Thanks to Gaberiel, this is now part of YTT setup for Sources

Let me remove

Reply:
Haha, ok. 🦅 👁️ can still do it 🤣

Reply:
Resolved